Gemfile

```diff
@@ -63,28 +27,14 @@ if RUBY_VERSION >= '1.9.3'
   end
 end
 
-# Rack::Cache 1.3.0 requires Ruby >= 2.0.0
-gem 'rack-cache', '< 1.3.0' if RUBY_VERSION < '2.0.0'
-
-if RUBY_VERSION < '1.9.2'
-  gem 'nokogiri', '~> 1.5.0'
-elsif RUBY_VERSION < '1.9.3'
-  gem 'nokogiri', '1.5.2'
-elsif RUBY_VERSION < '2.1.0'
-  gem 'nokogiri', '1.6.8.1'
-elsif RUBY_VERSION < '2.3.0'
-  gem 'nokogiri', '1.8.5'
-else
-  gem 'nokogiri', '~> 1.10'
-end
+# Minitest version 5.12.0 rely on Ruby 2.4 features and doesn't specify a Ruby version constraint
+gem 'minitest', '!= 5.12.0'
 
-if RUBY_VERSION <= '1.8.7'
-  # cucumber and gherkin require rubyzip as a runtime dependency on 1.8.7
-  # Only < 1.0 supports 1.8.7
-  gem 'rubyzip', '< 1.0'
-elsif RUBY_VERSION < '2.4'
-  gem 'rubyzip', '>= 1.2.2', '< 2.0.0'
-end
+gem 'mime-types'
+gem 'nokogiri'
+gem 'rake'
+gem 'rack-cache'
+gem 'rubyzip'
```

Comment:
Are these needed directly? We mostly specified them to pin them, I don't think anything but `rake` is needed?

Reply:
Good point, thanks.